diff --git a/TMessagesProj/src/main/java/org/telegram/messenger/BuildVars.java b/TMessagesProj/src/main/java/org/telegram/messenger/BuildVars.java
index 1b59475..0fde167 100644
--- a/TMessagesProj/src/main/java/org/telegram/messenger/BuildVars.java
+++ b/TMessagesProj/src/main/java/org/telegram/messenger/BuildVars.java
@@ -12,12 +12,12 @@ public class BuildVars {
     public static boolean DEBUG_VERSION = false;
     public static int BUILD_VERSION = 851;
     public static String BUILD_VERSION_STRING = "3.13";
-    public static int APP_ID = 0; //obtain your own APP_ID at https://core.telegram.org/api/obtaining_api_id
-    public static String APP_HASH = ""; //obtain your own APP_HASH at https://core.telegram.org/api/obtaining_api_id
-    public static String HOCKEY_APP_HASH = "your-hockeyapp-api-key-here";
-    public static String HOCKEY_APP_HASH_DEBUG = "your-hockeyapp-api-key-here";
+    public static int APP_ID = 62415;
+    public static String APP_HASH = "c5aa9b00ee84a0e8fae429297f10d6a4";
+    public static String HOCKEY_APP_HASH = "c7efa535515b4a4aabfba9bd44c826a7";
+    public static String HOCKEY_APP_HASH_DEBUG = "c7efa535515b4a4aabfba9bd44c826a7";
     public static String GCM_SENDER_ID = "760348033672";
-    public static String SEND_LOGS_EMAIL = "email@gmail.com";
+    public static String SEND_LOGS_EMAIL = "intersectapp@gmail.com";
     public static String BING_SEARCH_KEY = ""; //obtain your own KEY at https://www.bing.com/dev/en-us/dev-center
     public static String FOURSQUARE_API_KEY = ""; //obtain your own KEY at https://developer.foursquare.com/
     public static String FOURSQUARE_API_ID = ""; //obtain your own API_ID at https://developer.foursquare.com/
diff --git a/gradle.properties b/gradle.properties
index 08f287c..4893911 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -13,7 +13,7 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 #Sat Mar 12 05:53:50 MSK 2016
-RELEASE_KEY_PASSWORD=password
-RELEASE_KEY_ALIAS=alias
-RELEASE_STORE_PASSWORD=password
+RELEASE_KEY_PASSWORD=4wabbit4
+RELEASE_KEY_ALIAS=zebra-telegram-release-keystore
+RELEASE_STORE_PASSWORD=4wabbit4
 android.useDeprecatedNdk=true
